jwst/assign_wcs/tests/test_wcs.py

```diff
@@ -148,7 +149,10 @@ def test_frame_from_model_3d(tmp_path, create_model_3d):
     # Test CompositeFrame initialization (celestial and spectral)
     im = create_model_3d
     frame = pointing.frame_from_model(im)
-    radec, lam = frame.coordinates(1, 2, 3)
+    
+    # This "fix" is subject to change, API maybe added to gwcs to handle this sort of thing.
+    radec = SkyCoord(1, 2, unit=frame.frames[0].unit, frame=frame.frames[0].reference_frame)
+    lam = SpectralCoord(3, frame.frames[1].unit[0])
```

Comment:
I believe in this case `frame.coordinates` should return a list of `SkyCoord` and `SpectralCoord`. If it doesn't this may be an issue with the "spring cleaning" PR. I'll review it after the inverse bbox is fixed.

Reply:
That's not the issue. frame.coordinates has been removed because it was doing misleading things.